CHANGELOG.md

```diff
@@ -1,2 +1,4 @@
 ## Draft theme template 1.x.x
-* [Issue 1](https://github.com/lemberg/draft-theme-template/issues/1) — create Draft starter theme
\ No newline at end of file
+
+- [Issue 1](https://github.com/lemberg/draft-theme-template/issues/1) — create Draft starter theme
+- [Issue 4](https://github.com/lemberg/draft-theme-template/issues/4) — setup Webpack scaffolding
```

Comment:
New items in list of changes need add on the start.

Reply:
Done